tests/session_ut/session_ut.c

```diff
@@ -156,8 +156,8 @@ TEST_FUNCTION_CLEANUP(method_cleanup)
 
 /* session_create */
 
-/* Tests_SRS_SESSION_01_030: [session_create shall create a new session instance and return a non-NULL handle to it.] */
-/* Tests_SRS_SESSION_01_032: [session_create shall create a new session endpoint by calling connection_create_endpoint.] */
+/* Tests_S_R_S_SESSION_01_030: [session_create shall create a new session instance and return a non-NULL handle to it.] */
```

Comment:
>/* Tests_S_R_S_SESSION_01_030 [](start = 0, length = 29)

Ensure there is an issue to enable traceablity for these.

Reply:
There are 2 issues to enable the tests, one for connection, one for session. Thanks!

---
In reply to: [194185776](https://github.com/Azure/azure-uamqp-c/pull/242#discussion_r194185776) [](ancestors = 194185776)